src/features/investor-input/InvestorFields.tsx

```diff
@@ -67,25 +68,29 @@ export const Email = () => {
 
 export const FirstName = () => {
   const props = useController(useContext().focus('firstName').interop());
+  const type: InvestorType = useWatch({ name: 'investor.type' }) as InvestorType;
```

Comment:
I believe you should be able to pull the field control & TS type from the "investor" lens (which is what `useContext` holds)
```suggestion
  const type: InvestorType = useWatch(useContext().focus('type').interop());
```